src/api/app/models/staging_project.rb

```diff
@@ -0,0 +1,142 @@
+# The staging project is the place where we collect a selection of packages and test, build and review them.
+class StagingProject < Project
+  has_many :staged_requests, class_name: 'BsRequest', foreign_key: :staging_project_id, dependent: :nullify
+  has_many :status_reports, through: :repositories
+  belongs_to :staging_workflow, inverse_of: :staging_projects
+
+  def staging_identifier
+    name[/.*:Staging:(.*)/, 1]
+  end
+
+  def untracked_requests
+    requests_to_review - staged_requests
```

Comment:
I would really like to prefer to use proper ActiveRecord collections instead off doing these array calculations. In https://github.com/openSUSE/open-build-service/pull/6190 @vpereira and me actually needed to refactor some of this stuff introduced earlier because it was just to difficult to work with arrays instead of an ActiveRecord collection ...

I think a simple solution like this could even work:

```
requests_to_review.where.not(staged_requests)
```

Reply:
'I would really like to prefer'? How polite you are :)

Reply:
@coolo thank my parents to raise such a polity young boy :smile: :family_man_woman_boy:

Reply:
`requests_to_review` currently returns an Array. We have some follow up commits that replaces the existing code with scopes. Once this is merged I would open another PR and update the `untracked_requests` method as well.

Reply:
> ChrisBr 21 minutes ago  Member
> @coolo thank my parents to raise such a polity young boy 

The only task left for them is to make you eat your vegetables:laughing:

Reply:
> requests_to_review currently returns an Array. We have some follow up commits that replaces the existing code with scopes. Once this is merged I would open another PR and update the untracked_requests method as well.

Ok but we should do this ASAP as we really run into some problems in the mentioned PR...